study/Cloud Computing/k8s-files/keycloak/pod.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: postgres-pod
+  labels:
+    app: postgres-pod
+spec:
+  containers:
+    - name: postgres
+      image: postgres:latest
+      env:
+        - name: POSTGRES_USER
+          value: "keycloak"
+        - name: POSTGRES_PASSWORD
+          value: "mypassword"
+        - name: POSTGRES_DB
+          value: "keycloak"
+      ports:
+        - containerPort: 5432
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: keycloak-pod
+  labels:
+    app: keycloak-pod
+spec:
+  containers:
+    - name: keycloak
+      image: custom-keycloak:latest
+      imagePullPolicy: IfNotPresent # Use local image if available
+      env:
+        - name: KC_DB
+          value: "postgres"
+        - name: KC_DB_URL
+          value: "jdbc:postgresql://postgres-service:5432/keycloak"
+        - name: KC_DB_USERNAME
+          value: "keycloak"
+        - name: KC_DB_PASSWORD
+          value: "mypassword"
+        - name: KC_BOOTSTRAP_ADMIN_USERNAME
+          value: "admin"
+        - name: KC_BOOTSTRAP_ADMIN_PASSWORD
+          value: "admin"
+        - name: KC_HOSTNAME
+          value: "localhost"
```

Comment:
Same problem as commented before please check the review in the Dockerfile of Keycloak